packages/node_modules/@ciscospark/plugin-conversation/src/share-activity.js

```diff
@@ -333,8 +333,8 @@ ShareActivity.create = function create(conversation, object, spark) {
     parent: spark
   });
 
-  if (files) {
-    files.forEach((file) => share.add(file));
+  if (files && files.items) {
+    files.items.forEach((file) => share.add(file));
```

Comment:
right bzang, The activities returned from conversation service has files.items. Do you suggest any changes for backward compatibility or are we good with this PR?